.github/workflows/e2e.yml

```diff
@@ -46,8 +46,8 @@ jobs:
               echo "DFX_NO_WALLET=--no-wallet" >> "$GITHUB_ENV"
           fi
 
-      - name: 'Run tests'
-        run: bats e2e/bash/icx-asset.bash e2e/bash/icx-asset-upload.bash
+      # - name: 'Run tests'
+      #   run: bats e2e/bash/icx.bash 
```

Comment:
while fixing things for this PR, I've noticed `icx` e2e tests were never activated in GHA workflow. I tried activating them, however, there were errors, so I decided it's better to fix that in dedicated PR.